src/index.html

```diff
@@ -10,13 +10,56 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+
     <title>Moyo header</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
-  <body>
-    <h1>Moyo header</h1>
-  </body>
+  <header>
+    <a
+      href="null"
+      class="logo"
+    >
+      <img
+        src="https://s3-alpha-sig.figma.com/img/833f/9aab/47d660efd384318772b6b8040db831aa?Expires=1725840000&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=MJRW8Yu0t9rfw~RbBcB~YImlgu22Gqhw2VLCvrJt9QpHubYRZwarK8rWfd0cXYEq3UdLnf2pTYuHtGkIMDr2rGTTba0KbaiAY4iTHQLsAoyHlo~eXIBi-CbDvG1aArEN2ybS2cDYwwOLuHm78zORnVD8AIhelkjjsme9N7SqEsADP1J0Kh0aKujhSICCib5MWOXbMHON4vv8ZK2qmlpdH8dJdm1h-Ahcsvw3pRFygjDfC9e0kZmk1ogcKrzrPTnxQbUeclnN-budN5yWzdz2icxTPVf7qdE98Sx9nKSkHLTXcbRKQZlEdqYer8R2A6sL4FoBsP1KL2IVJuFVP9v83w__"
+        alt="noyo_logo"
+      />
+    </a>
+    <nav>
+      <ul class="nav-flex">
+        <li>
```

Comment:
Fix classes everywhere
```suggestion

    <nav class="nav">
      <ul class="nav__list">
        <li class="nav__item">
```